lib/msal-common/src/utils/Constants.ts

```diff
@@ -389,3 +389,8 @@ export const ONE_DAY_IN_MS = 86400000;
 
 // Token renewal offset default in seconds
 export const DEFAULT_TOKEN_RENEWAL_OFFSET_SEC = 300;
+
+export const EncodingTypes = {
+    HEX: "hex",
```

Comment:
I'll be making another PR where I replace every instance of "hex" with this new constant, and make "base64" value as well.

Reply:
I would recommend to create a github issue and link it here for reference

Reply:
done: https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/7765